hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsPermission.java

```diff
@@ -68,7 +68,7 @@ public boolean equals(Object obj) {
    * @return a permission object for the provided string representation
    */
   public static AbfsPermission valueOf(final String abfsSymbolicPermission) {
-    if (abfsSymbolicPermission == null) {
+    if (abfsSymbolicPermission == null || abfsSymbolicPermission.isEmpty()) {
```

Comment:
StringUtils.isEmpty(abfsSymbolicPermission) would do the same work.

Reply:
Thanks for the suggestion, taken